src/main/java/tatracker/model/TaTracker.java

```diff
@@ -132,11 +137,28 @@ public void setSessions(List<Session> sessions) {
      */
     public void addDoneSession(Session s) {
         doneSessions.add(s);
-        totalHours += Math.ceil(s.getDuration().toHours());
     }
 
-    public static long getTotalEarnings() {
-        return rate * totalHours;
+    @Override
+    public long getTotalHours() {
+        return doneSessions.asUnmodifiableObservableList()
+                .stream()
+                .map(s -> (long) Math.ceil(s.getDuration().toHours()))
+                .reduce(DEFAULT_HOURS, Long::sum);
+    }
```

Comment:
![image](https://user-images.githubusercontent.com/42372470/78224662-7febe200-74fb-11ea-8a40-bd474bdac69f.png)
the Math.ceil() doesn't work properly here. Should have one extra hour for each session, but its round down instead. Perhaps you can remove the to.Hours() function? lmk when you're done and ill give it a try again